.ci/azure-pipelines/build_windows.bat

```diff
@@ -83,5 +83,5 @@ copy installer\client\mumble_client*.msi %BUILD_ARTIFACTSTAGINGDIRECTORY%
 
 copy installer\server\mumble_server*.msi %BUILD_ARTIFACTSTAGINGDIRECTORY%
 
-7z a PDBs.7z *.pdb
+7z a PDBs.7z *.pdb plugins\*
```

Comment:
Doesn't this also include all the actual libraries and sub-directories cmake created?

Reply:
Yes, sorry.